<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>53</y>
     <width>420</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>130</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>보내는 사람</string>
   </property>
   <widget class="QLineEdit" name="lineSenderAddress">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>95</y>
      <width>330</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSenderName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>33</y>
      <width>330</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSenderNumber">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>64</y>
      <width>330</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelSenderAddress">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>91</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRecevierName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>33</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSenderNumber">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>62</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전화번호</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>192</y>
     <width>420</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>130</height>
    </size>
   </property>
   <property name="title">
    <string>받는 사람</string>
   </property>
   <widget class="QLineEdit" name="lineReceiverrNumber">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>64</y>
      <width>330</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineReceiverName">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>33</y>
      <width>330</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineReceiverAddress">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>95</y>
      <width>330</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelReceiverAddress">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>91</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelReceiverName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>33</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelReceiverNumber">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>62</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전화번호</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>340</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnRead">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>읽기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnWrite">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>입력</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDownload">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>불러오기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUpload">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>저장하기</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineProductName">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>21</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>330</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>42</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>상품명</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelState">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
